src/lib/components/primitives/message/input-message.stories.js

```diff
@@ -0,0 +1,25 @@
+import ErrorComponent from './input-message.svelte';
+
+export default {
+  component: ErrorComponent,
+  parameters: {
+    layout: 'centered',
+  },
+  title: 'Primitives/InputMessage',
+};
+
+export const Base = {
+  args: {
+    message: 'Please review your input.',
+    key: 'infoMessage',
+    type: 'info'
+  },
+};
+
+export const Error = {
+  args: {
+    message: 'Please review your input.',
+    key: 'infoMessage',
+    type: 'info'
```

Comment:
Darn it! This should be type "error". I'll fix this along with any feedback I get from you all.